CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 ## 3.56.0 (Unreleased)
 
+NOTES:
+* billingBudget: doc updated for `google_billing_budget` ([#8316](https://github.com/hashicorp/terraform-provider-google/pull/8316))
```

Comment:
I don't think this is worth adding to the changelog, unless it was going to call out something important that was added to the documentation. My bad for not catching it
```suggestion
```